src/inmanta/ast/statements/__init__.py

```diff
@@ -234,7 +252,7 @@ def get_dataflow_node(self, graph: DataflowGraph) -> dataflow.NodeReference:
         raise NotImplementedError()
 
 
-class Resumer(ExpressionStatement):
+class Resumer(Locatable):
```

Comment:
I didn't get any type errors when I did this and I see no reason why it should be an `ExpressionStatement`. If you think differently, let me know.